src/main/java/org/alliancegenome/curation_api/services/helpers/UniqueIdentifierHelper.java

```diff
@@ -0,0 +1,72 @@
+package org.alliancegenome.curation_api.services.helpers;
+
+import org.alliancegenome.curation_api.model.entities.Annotation;
+import org.alliancegenome.curation_api.model.entities.base.AuditedObject;
+import org.alliancegenome.curation_api.model.entities.base.SubmittedObject;
+import org.alliancegenome.curation_api.model.ingest.dto.AnnotationDTO;
+import org.alliancegenome.curation_api.model.ingest.dto.DiseaseAnnotationDTO;
+import org.alliancegenome.curation_api.model.ingest.dto.base.SubmittedObjectDTO;
+import org.apache.commons.lang3.StringUtils;
+
+public class UniqueIdentifierHelper {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.design.HideUtilityClassConstructorCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Utility classes should not have a public or default constructor.
<!-- __reviewdog__:ChA4ZmIzMzU1MDQ0NjhjZmViEgpjaGVja3N0eWxl -->